### PR TITLE
Get File Sample By Hash - Generic v3 - updated description to include Code42

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Related to: https://jira-dc.paloaltonetworks.com/browse/CIAC-8220

## Description
Updated the playbook description and README file to include the usage of Code42.

### CODE REVIEW

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Get File Sample By Hash - Generic v3
+
+- Updated the playbook description to reflect the addition of the Code42 sub-playbook.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md:6]:
```suggestion
Updated the playbook description to reflect the addition of the Code42 sub-playbook.
```

**Comment by ShirleyDenkberg** [comment]
@michalgold @AdiPeret Doc review completed.

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml [hunk 2]
```diff
@@ -81,7 +87,6 @@ tasks:
     scriptarguments:
       SHA256:
         simple: ${inputs.SHA256}
```
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml:89]: old value but fixing

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml [hunk 3]
```diff
@@ -108,8 +116,7 @@ tasks:
       id: 3ec72fe1-247c-4230-8f59-4259ecdca49a
       version: -1
       name: Get the binary file from Carbon Black by its MD5 hash
-      description: This playbook retrieves a binary file by its MD5 hash from the
-        Carbon Black telemetry data.
+      description: This playbook retrieves a binary file by its MD5 hash from the Carbon Black telemetry data.
       playbookName: Get the binary file from Carbon Black by its MD5 hash
       type: playbook
       iscommand: false
```
[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml:125]: image
@idovandijk - pls add here unique transformer as well. I know that's an old value, but still let's make it right.

**Review by ssokolovich — APPROVED** [review]

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml [hunk 4]
```diff
@@ -253,24 +263,7 @@ outputs:
...
-- Detonate File - No Files test
-- Domain Enrichment - Generic v2 - Test
-- Calculate Severity - Standard - Test
-- Retrieve File from Endpoint - Generic V2 Test
-- Extract Indicators From File - Generic v2 - Test
-- Field polling test
-- Email Address Enrichment - Generic v2.1 - Test
-- File Enrichment - Generic v2 - Test
-- IP Enrichment - Generic v2 - Test
+- No tests (auto formatted)
```
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml:271]: Yeah these tests were wrongly collected by broken SDK versions. They're irrelevant for this flow

**Pull request merged**